DealerInspireLaravelCodingStandard/Sniffs/Models/NoGuardedAttributesSniff.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace DealerInspireLaravelCodingStandard\Sniffs\Models;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * NoGuardedAttributesSniff is a class that detects when a class has a protected $guarded attribute.
+ *
+ * @package DealerInspireLaravel\Sniffs\Models
+ */
+class NoGuardedAttributesSniff implements Sniff
+{
+    /**
+     * Returns the token types that this sniff is interested in.
+     *
+     * @return int[]
+     */
+    public function register(): array
+    {
+        return [T_PROTECTED];
+    }
+
+    /**
+     * Processes this sniff, when one of its tokens is encountered.
+     *
+     * @param  File $phpcsFile The current file being checked.
+     * @param  mixed $stackPtr  The position of the current token in the stack passed in $tokens. Type of mixed due to the interface not enforcing int.
+     * @return void
+     */
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        $tokens = $phpcsFile->getTokens();
+
+        $variablePointer = $phpcsFile->findNext([T_VARIABLE], $stackPtr);
```

Comment:
🔥 `->findNext`

Reply:
Yeah, I really wanted to see if there was some way to avoid having variables and loops and reset stuff so I spent some time digging through some of the more simple Slevomat rules. They have a lot of helper functions that we don't have access to, but I found this gem buried in there somewhere. Really nice when you use something that _has_ to have a specific token after it eventually.